indico/web/assets/vars_js.py

```diff
@@ -122,6 +123,14 @@ def generate_global_file(config):
                         'manage': url_rule_to_js('timetable.entry_info_manage'),
                     },
                 }
+            },
+
+            'Contributions': {
+                'display_contribution': url_rule_to_js('contributions.display_contribution')
```

Comment:
I thinks it's less confusing if we use the same names and structure as in the blueprints.